### ISSUE TITLE
Project is not compiling

### ISSUE THREAD

**Issue by Shiaulis** [OpenCovidTrace/octrace-ios#14]
I didnt' run project for several weeks, but today I totally failed to compile on any commit :( 
The error I receive is related to `DP3TSDK` project:
![image](https://user-images.githubusercontent.com/13601748/82669079-6d607000-9c43-11ea-849c-e10f985560e4.png)

I did different attempts fo fix this but without any success.
The only thing I didn't' check is that it can be connected to Xcode update to version 11.5. But maybe you know any other solutions? Just in case I'll download the old version just to check did it worked before or not

**Comment by alexnivanov** [comment]
That's weird, I'm using Xcode 11.4.1 though and it's working fine.

**Comment by Shiaulis** [comment]
Yep, will check later today. Just in case I would suggest not update Xcode 🙂

**Comment by alexnivanov** [comment]
Ha-ha yeah, that's why Xcode 11.5 is still beta ;)

**Comment by Shiaulis** [comment]
Why do you think so? 🙂 check AppStore version of Xcode 🙂

**Comment by alexnivanov** [comment]
True!

Well, that's what google told me:

![image](https://user-images.githubusercontent.com/13519834/82678679-aeac4c00-9c52-11ea-8199-51e51f9af610.png)

Still good that I didn't update yet 😅

**Comment by Shiaulis** [comment]
Ah google… 😅

**Comment by alexnivanov** [comment]
Yeah well I actually started updating Xcode, cuz I'm leaving to country side for a week where the internet is too slow... So we'll see how it goes :)

**Comment by alexnivanov** [comment]
Works for me:

Xcode Version 11.5 (11E608c)

Try clean & remove derived data.

**Comment by Shiaulis** [comment]
🤦 Dunno what else can be the reason. I even delete and download repo, didn't help. Thanks anyway, will investigate more what can be the reason

**Comment by Shiaulis** [comment]
I tried on another machine and have the same issue. I will close the issue but just keep in mind I won't be able to compile project until I find a solution

**Comment by alexnivanov** [comment]
That's weird, we'll try with other team members.

**Comment by Shiaulis** [comment]
it would be great. Hope to continue to invest my time in your app :)

**Comment by Shiaulis** [comment]
https://github.com/DP-3T/dp3t-sdk-ios/issues/108 <- the same issue from other devs so hope it will be fixed soon
